tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1221,6 +1221,88 @@ some text
 
 `;
 
+exports[`inline2.php 1`] = `
+<?php
+// test
+?>
+<h1>Test</h1>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+// test
+?>
+<h1>Test</h1>
+
+`;
+
+exports[`inline3.php 1`] = `
+<h1>Test</h1>
+<?php
+// test
+?>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<h1>Test</h1>
+<?php
+// test
+?>
+`;
+
+exports[`inline4.php 1`] = `
+<?php
+/**
+ * A
+ */
+
+// B
+
+/**
+ * C
+ */
+?>
+<h1>Test</h1>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<?php
+/**
+ * A
+ */
+// B
```

Comment:
Still need to preserve linebreaks

Reply:
@czosel please create issue too for this :+1:

Reply:
https://github.com/prettier/plugin-php/issues/506